MdeModulePkg/Library/UefiVariablePolicyLib/UefiVariablePolicyUnitTest/UefiVariablePolicyUnitTest.c

```diff
@@ -345,25 +385,34 @@ WildcardPoliciesShouldMatchDigits (
   CHAR16        *CheckVar1Name = L"Wildcard1VarName12";
   CHAR16        *CheckVar2Name = L"Wildcard2VarName34";
   CHAR16        *CheckVarBName = L"WildcardBVarName56";
+  CHAR16        *CheckVarFName = L"WildcardFVarName0A";
+  CHAR16        *CheckVarZName = L"WildcardZVarName56";
```

Comment:
aren't there three duplicates with Name56?  

Should you include a few more like a, z, and Z.  Just to make sure you exercise all the conditionals.

Reply:
Sure thing.

Reply:
Though, I'll have to remember not to drop the new test cases if/when this lands in EDK2 (since the patches don't currently have them (or the docs, for that matter)).